projects/giflib/dgif_target.options

```diff
@@ -0,0 +1,3 @@
+[libfuzzer]
+close_fd_mask = 3
```

Comment:
Is this needed? I've just run the fuzz target and it doesn't seem to log anything.